src/framework/MOM_io.F90

```diff
@@ -2233,7 +2316,80 @@ subroutine MOM_io_init(param_file)
   call log_version(param_file, mdl, version)
 
 end subroutine MOM_io_init
-
+!> Returns the dimension variable information for a netCDF variable
+subroutine get_var_axes_info(filename, fieldname, axes_info)
+  character(len=*), intent(in) ::            filename  !< A filename from which to read
+  character(len=*), intent(in) ::            fieldname !< The name of the field to read
+  type(axis_info), dimension(4), intent(inout) :: axes_info !< A returned array of field axis information
+
+  !! local variables
+  real ::  rcode
```

Comment:
The return code from all NF90 calls are integers, not reals, so this line should be `integer :: rcode`.